src/fakeprinter/fakeprinter/middlewares

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Missing .py extension

Reply:
fixed in 497638cc030d6e140d703934e0feb7120e1154b5